CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+`1.5.1`_ - 2021-01-25
+---------------------
+
+**Features**
+
+* New parameter called `CUSTOM_FAILED_RESPONSE_VIEW`, allowing you to set a custom django function view to handle login
+  failures. #?
```

Comment:
```suggestion
  failures. #136
```

Reply:
@peterfarrel, I'm unable to write to your branch. Could you just add this as well before I merge?